src/elements/Promise.test.tsx

```diff
@@ -0,0 +1,26 @@
+/* @jsx MD */
+import MD, { render } from "..";
+
+import { Text, MdPromise } from ".";
+
+describe("MdPromise", () => {
+  it("renders a single promise that resolves to a string", async () => {
+    expect(
+      await render(<MdPromise>{new Promise((res) => res("Test"))}</MdPromise>)
```

Comment:
Please use Promise.resolve and Promise.reject in this file

Reply:
Noted.